pp/integrations/plugin-packs/getting-started/how-to-guides/cma/cma-setup.md

```diff
@@ -644,6 +721,21 @@ apt -y install centreon-plugin-operatingsystems-linux-local
 </TabItem>
 </Tabs>
 
+### Updating an existing configuration
+
+<Tabs groupId="sync">
+<TabItem value="Linux" label="Linux">
+Modify the **/etc/centreon-monitoring-agent/centagent.json** file and restart the agent.
+</TabItem>
+<TabItem value="Windows" label="Windows">
+Run **centreon-monitoring-agent-modify.exe** located in the CMA installation directory.
+This can also be done in silent mode
+```shell
+centreon-monitoring-agent-modify.exe /VERYSILENT
+```
```

Comment:
```suggestion
<Tabs groupId="sync">
<TabItem value="Linux" label="Linux">

Modify the **/etc/centreon-monitoring-agent/centagent.json** file and restart the agent.

</TabItem>
<TabItem value="Windows" label="Windows">

Run **centreon-monitoring-agent-modify.exe** located in the CMA installation directory.
This can also be done in silent mode
```shell
centreon-monitoring-agent-modify.exe /VERYSILENT
```

```